content/en/docs/contribute/localization.md

```diff
@@ -13,53 +13,66 @@ card:
-
-{{< note >}}
-For an example localization-related [pull request](../create-pull-request), see [this pull request](https://github.com/kubernetes/website/pull/8636) to the [Kubernetes website repo](https://github.com/kubernetes/website) adding Korean localization to the Kubernetes docs.
-{{< /note >}}
+All localization teams must be self-sustaining with their own resources. We're happy to host your work, but we can't translate it for you.
 
-Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). We're happy to help you get started and answer any questions you have.
+### Find your two-letter language code
 
-All localization teams must be self-sustaining with their own resources. We're happy to host your work, but we can't translate it for you.
+First, consult the [ISO 639-1 standard](https://www.loc.gov/standards/iso639-2/php/code_list.php) to find your localization's two-letter country code. For example, the two-letter code for Korean is `ko`.
 
 ### Fork and clone the repo
 
-First, [create your own fork](https://help.github.com/articles/fork-a-repo/) of the [kubernetes/website](https://github.com/kubernetes/website).
+First, [create your own fork](/docs/contribute/start/#improve-existing-content) of the [kubernetes/website](https://github.com/kubernetes/website) repository.
 
-Then, clone the website repo and `cd` into it:
+Then, clone your fork and `cd` into it:
 
 ```shell
-git clone https://github.com/kubernetes/website
+git clone https://github.com/<username>/website
 cd website
 ```
 
-{{< note >}}
-Contributors to `k/website` must [create a fork](/docs/contribute/start/#improve-existing-content) from which to open pull requests. For localizations, we ask additionally that:
+### Open a pull request
 
-1. Team approvers open development branches directly from https://github.com/kubernetes/website.
-2. Localization contributors work from forks, with branches based on the current development branch.
+Next, [open a pull request](https://kubernetes.io/docs/contribute/start/#submit-a-pull-request) (PR) to add a localization to the `kubernetes/website` repository. 
 
-This is because localization projects are collaborative efforts on long-running branches, similar to the development branches for the Kubernetes release cycle. For information about localization pull requests, see ["branching strategy"](#branching-strategy).
-{{< /note >}}
+The PR must include all of the [minimum required content](#minimum-required-content) before it can be approved.
 
-### Find your two-letter language code
+For an example of adding a new localization, see the PR to enable [docs in French](https://github.com/kubernetes/website/pull/8636).
```

Comment:
The URL links to the initialization of the Korean l10n. We should mention `docs in Korean` or update the URL to point to #12548